pkgs/top-level/perl-packages.nix

```diff
@@ -18419,8 +18419,8 @@ let
   SysVirt = buildPerlModule rec {
     pname = "Sys-Virt";
     version = "6.3.0";
-    src = assert version == pkgs.libvirt.version; fetchgit {
-      url = "git://libvirt.org/libvirt-perl.git";
+    src = fetchgit {
+      url = "https://gitlab.com/libvirt/libvirt-perl.git";
```

Comment:
Can this use `fetchFromGitLab`?

Reply:
TIL about `fetchFromGitLab`. I'll replace it.  Thanks.